### ISSUE TITLE
select() data type not applied when $columns is a string

### ISSUE THREAD

**Issue by mgibbs189** [catfan/Medoo#880]
@catfan Thanks for the great library, much appreciated.

Similar to #859, the data type isn't enforced in `select()` when `$columns` is a string, but _**is**_ enforced when it's an array.

```php
$db->select( 'users', 'id[Int]', [ 'LIMIT' => 1 ] );
// returns "1" (incorrectly set as string)
```

and

```php
$db->select( 'users', [ 'id[Int]' ], [ 'LIMIT' => 1 ] );
// returns [ 'id' => 1 ] (correctly set as an integer)
```

**Comment by catfan** [comment]
This fixed it on 861750a.

**Comment by mgibbs189** [comment]
Excellent, thanks! 👍
